charts/gitops-server/templates/deployment.yaml

```diff
@@ -36,6 +40,21 @@ spec:
             - "{{ .Release.Namespace }}"
             - "--log-level"
             - "{{ .Values.logLevel }}"
+            {{- if .Values.serverTLS.useSecret }}
+            - "--tls-cert-file"
+            - "/etc/tls-volume/{{ .Values.serverTLS.certContentField }}"
+            - "--tls-private-key-file"
+            - "/etc/tls-volume/{{ .Values.serverTLS.keyContentField }}"
+            {{- end }}
+            {{- if .Values.serverTLS.useLocalFiles  }}
```

Comment:
This should be `{{- else if....` otherwise it will be possible to set multiple `--tls-private-key-file` flags and `--insecure`